appveyor_after_build.ps1

```diff
@@ -0,0 +1,300 @@
+. .\appveyor_helpers.ps1
+
+####################### Local run (Debug) configuration ###################################
+
+
+# Set this to true to be able to run outside of appveyor - just configure the section below
+$LOCAL_DEBUG_RUN = $false
+
+
+# Set these input variable that would normally come from AppVeyor yml
+if ($LOCAL_DEBUG_RUN)
+{    
+    $env:SQ_Version = "4.5.5"   
+    $env:APPVEYOR_BUILD_FOLDER = "C:\Users\bgavril\source\repos\sonar-msbuild-runner"
+    $env:configuration = "Debug"
+
+    # This script does not build the C# plugin (unlike appveyor) - please provide the path to it for the debug run
+    $localCSharpPluginPath = $env:APPVEYOR_BUILD_FOLDER + "\DeploymentArtifacts\sonar-csharp-plugin-4.2.Debug.bgavril.jar";
```

Comment:
The before_script will have build the C# plugin. Appveyor will have build the SQ Msbuild Scanner, including the packaging project.